net-im/slack++/slack++-0.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Enterprise chat and services on the command line - slack++"
+DESCRIPTION="slack++ is an ncurses based client the Searchable Log of All Conversation and Knowledge"
+HOMEPAGE="https://github.com/ga2arch/slack"
+COPYRIGHT="2018"
+LICENSE="GNU GPL v2"
```

Comment:
I'm actually a bit skeptical about this. None of the source files state what they are licensed as, there is no COPYRIGHT/LICENSE file, only an `PKGBUILD` with the `licenses` field set to `GPL`

Reply:
I fail to see why, a copyright or license file is a convention not a mandate.

The PKGBUILD is doing the same thing.

Reply:
Asked for clarification here https://github.com/ga2arch/slack/issues/19

Reply:
Got feedback and fixed this the source is now GPL v3 explicitly.